lib/screens/profile/profile.dart

```diff
@@ -196,6 +196,10 @@ class _ProfileState extends ConsumerState<ProfileScreen> {
                                 },
                               )
                             ] else if (FlavorConfig.current.flavor != Flavor.kids) ...[
+                              OptionButton(
```

Comment:
I missed to move the button to the right place. Should this button be shown in all cases? i.e  authenticated, guests, other flavours?